control_panel_api/settings.py

```diff
@@ -110,7 +111,7 @@
 
 REST_FRAMEWORK = {
     'DEFAULT_PERMISSION_CLASSES': [
-        'rest_framework.permissions.IsAdminUser',
+        'rest_framework.permissions.IsAuthenticated',
```

Comment:
I'd probably stick with `IsAdminUser` as `IsAuthenticated` is quite permissive (I know this will change anyway).